src/Listener/ConsoleListener.php

```diff
@@ -46,10 +47,15 @@ public function onCommand(BaseConsoleEvent $e)
     }
 
     /**
-     * @param BaseConsoleEvent $e
+     * @param ConsoleTerminateEvent $e
      */
-    public function onTerminate(BaseConsoleEvent $e)
+    public function onTerminate(ConsoleTerminateEvent $e)
```

Comment:
Here is a possible compatibility break, but as Symfony always give this Event class in all Symfony 2.3+ versions, i have considered as a non-BC break update.